src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -170,11 +182,16 @@
                 return i18next.t('shareProjectInstructions', {project: this.project.name, lng: this.currentLanguage});
             },
             invalidEmails() {
+                //Invalid state until all emails evaluated as valid = true (may not have blurred yet)...
                 return this.emailsToShareWith.filter(e => e.value && !e.valid).length > 0
             },
+            showValidationMessage() {
+                //...however only show error message if any confirmed to be valid = false
+                return this.emailsToShareWith.filter(e => e.value && e.valid === false).length > 0
```

Comment:
So presumably this excludes valid values that are null.

Reply:
Exactly - they're the ones that haven't been validated yet